### PR TITLE
add jQuery-flexImages@1.0.4 w/ git auto-update

### LINKED ISSUE #8533: [Request] Add jQuery-flexImages
**Library name:** jQuery-flexImages
**Git repository url:** https://github.com/Pixabay/jQuery-flexImages

### PR DESCRIPTION
**Pull request by x09326**
PR for #8533 
@pvnr0082t please help me review it, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/Pixabay/jQuery-flexImages
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars:122
   - Count of watchers:8
   - Count of forks:34
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8533, cc @SimonSteinberger

### CODE REVIEW

File: ajax/libs/jQuery-flexImages/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "type": "git",
+    "url": "git://github.com/Pixabay/jQuery-flexImages.git"
+  },
+  "authors": [
+    {
+      "name": "Simon Steinberger",
+      "url": "https://pixabay.com/users/Simon/",
+      "email": "simon@pixabay.com"
+    }
+  ],
```
Comment (pvnr0082t) [inline ajax/libs/jQuery-flexImages/package.json]: @x09326 We can just use an object and remove array.

**Comment by pvnr0082t** [comment]
@PeterDaveHello The latest version we can get from https://goodies.pixabay.com/jquery/flex-images/demo.html is v1.0.4 but the latest version on GitHub is v1.0.1. Is it better to add v1.0.4?

**Comment by x09326** [comment]
I think they forget to update the latest version.
I will confirm that with them.

**Comment by PeterDaveHello** [comment]
@x09326 can you manually add the tagged versions and v1.0.4 first?  Thanks.

**Comment by x09326** [comment]
@pvnr0082t 
I have added the tagged versions.
Please help me review it, thanks.

**Comment by pvnr0082t** [comment]
@x09326 I think https://goodies.pixabay.com/jquery/flex-images/demo.html can be `homepage`.

**Comment by pvnr0082t** [comment]
The latest minify tool will not minify `jquery.flex-images.css` in v1.04 because there are only 3 lines. I think you can remove `jquery.flex-images.min.css` and `jquery.flex-images.min.css.map`

**Comment by x09326** [comment]
@pvnr0082t 
done!

**Comment by pvnr0082t** [comment]
@x09326 Please also mention related issue in second commit. Thanks.

**Comment by x09326** [comment]
@pvnr0082t 
I haved modified the commit message.
Please help me review it, thanks.

**Comment by pvnr0082t** [comment]
LGTM
But multi-commit PR need multi reviewer.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
@PeterDaveHello 
Should we also need to add old versions in this case?
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh your question is not clear at all, I have no idea what the situation you are talking about, would you please give some details?

@x09326 I found that the version v1.0.2/1.0.3 are mentioned in the readme but not tagged, would you please trace its commit log and manually add them? Thanks.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
Because it has git tag in `v1.0.0` and `v1.0.1`, I think it can be automatically added.
That's what I mean.
But there has old version `v1.0.2` and `v1.0.3` without tag you mentioned, it should be added manually now.
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh Yes, only manually add v1.0.0/1.0.1 here makes no sense.

**Comment by kennynaoh** [comment]
OK

**Comment by x09326** [comment]
@kennynaoh @pvnr0082t 
I have added ```1.0.2``` and ```1.0.3``` by tracing commit log.
Please help me review it, thanks.

**Comment by kennynaoh** [comment]
LGTM

**Comment by pvnr0082t** [comment]
@PeterDaveHello The files added by tracing commit log should be the last change? I mean, for example, https://github.com/Pixabay/jQuery-flexImages/commit/eb83790b1826aa6d51939e29dbbfb8d2865fce24 and https://github.com/Pixabay/jQuery-flexImages/commit/7757cd2f1168302dc929f5caf6eab2db87ab1f93 are different, but it seems like @x09326 add the latter(older) one.

**Comment by PeterDaveHello** [comment]
@pvnr0082t Nope, shuold be the first change tagged that version, thanks.

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**